support/filterlock/src/main.rs

```diff
@@ -0,0 +1,70 @@
+use std::{fs::File, env::{args_os, args}, io::Read, collections::BTreeSet};
```

Comment:
Missing a license here, I think, as well.

Reply:
done in 6f76a10fca2619f66182077031bc3430a1a4ba18